pages/[user]/[type].tsx

```diff
@@ -228,57 +248,84 @@ export default function Type(props): Type {
   );
 }
 
-export const getServerSideProps: GetServerSideProps = async (context) => {
-  const user = await prisma.user.findFirst({
-    where: {
-      username: context.query.user.toLowerCase(),
-    },
-    select: {
-      id: true,
-      username: true,
-      name: true,
-      email: true,
-      bio: true,
-      avatar: true,
-      eventTypes: true,
-      startTime: true,
-      timeZone: true,
-      endTime: true,
-      weekStart: true,
-    },
-  });
+export async function getServerSideProps(context) {
+    const user = await prisma.user.findFirst({
+        where: {
+          username: context.query.user,
+        },
+        select: {
+            id: true,
+            username: true,
+            name: true,
+            email: true,
+            bio: true,
+            avatar: true,
+            eventTypes: true,
+            startTime: true,
+            timeZone: true,
+            endTime: true,
+            weekStart: true,
+        }
+    });
 
-  if (!user) {
-    return {
-      notFound: true,
-    };
-  }
+    const subEventTypes = user.eventTypes.map(el => {
+      if (el.startDate !== null) {
+        return {
+          startDate: el.startDate.toString(),
+          endDate: el.endDate.toString(),
+        }
+      }
+      else {
+        return { 
+          startDate: el.startDate,
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689307)
```suggestion
        startDate: el.startDate,
```